relay-server/src/actors/upstream.rs

```diff
@@ -246,21 +325,71 @@ impl UpstreamRelay {
             //   1. Have own connection pool for `store` requests
             //   2. Buffer up/queue/synchronize events before creating the request
             .wait_timeout(self.config.event_buffer_expiry())
-            .conn_timeout(self.config.http_connection_timeout())
             // This is the timeout after wait + connect.
             .timeout(self.config.http_timeout())
+            .conn_timeout(self.config.http_connection_timeout())
```

Comment:
_nit_: Move this to the previous location (under `wait_timeout`) again to reduce the diff.

Reply:
ok